runtime/Cpp/runtime/src/support/Guid.h

```diff
@@ -32,7 +32,9 @@
 #include <string>
 #include <vector>
 
-#ifdef GUID_ANDROID
+#include "antlr4-common.h"
```

Comment:
Why's that include necessary here? Builds are usually faster the fewer headers are included.

Reply:
If one were to include `support/Guid.h` directly, it would compile incorrectly as the macros would not be defined. This resolves that by always ensuring the macros are defined.

Reply:
Another solution is to just move the macros to `support/Guid.h`, I'll leave that to you to decide.